core/libp2p/multi/hash_type.hpp

```diff
@@ -0,0 +1,45 @@
+//
+// Created by xappm on 01.03.19.
+//
+
+#ifndef KAGOME_HASH_TYPE_HPP
+#define KAGOME_HASH_TYPE_HPP
+
+namespace libp2p::multi {
+  enum HashType : uint64_t {
+    kBlake2s8 = 0xb241,
+    kBlake2s16 = 0xb242,
+    kBlake2s24 = 0xb243,
+    kBlake2s32 = 0xb244,
+    kBlake2s40 = 0xb245,
+    kBlake2s48 = 0xb246,
+    kBlake2s56 = 0xb247,
+    kBlake2s64 = 0xb248,
+    kBlake2s72 = 0xb249,
+    kBlake2s80 = 0xb24a,
+    kBlake2s88 = 0xb24b,
+    kBlake2s96 = 0xb24c,
+    kBlake2s104 = 0xb24d,
+    kBlake2s112 = 0xb24e,
+    kBlake2s120 = 0xb24f,
+    kBlake2s128 = 0xb250,
+    kBlake2s136 = 0xb251,
+    kBlake2s144 = 0xb252,
+    kBlake2s152 = 0xb253,
+    kBlake2s160 = 0xb254,
+    kBlake2s168 = 0xb255,
+    kBlake2s176 = 0xb256,
+    kBlake2s184 = 0xb257,
+    kBlake2s192 = 0xb258,
+    kBlake2s200 = 0xb259,
+    kBlake2s208 = 0xb25a,
+    kBlake2s216 = 0xb25b,
+    kBlake2s224 = 0xb25c,
+    kBlake2s232 = 0xb25d,
+    kBlake2s240 = 0xb25e,
+    kBlake2s248 = 0xb25f,
+    kBlake2s256 = 0xb260
```

Comment:
I am not sure that we need all those versions of Blake :)
We need 
Blake{128,256}

Other versions do not exist